app/index.html

```diff
@@ -11,23 +11,85 @@
 </head>
 <body>
     <section class="container connection">
-        <p class="not-supported">Votre navigateur ne supporte pas les WebComponents et/ou les modules JS natifs, utilisés par le FkBoard. Pour l'utiliser, merci de mettre à jour votre navigateur.</p>
+        <p class="not-supported">
+            Votre navigateur ne supporte pas les WebComponents et/ou les modules
+            JS natifs, utilisés par le FkBoard. Pour l'utiliser, merci de mettre
+            à jour votre navigateur.
+        </p>
         <form id="connection" method="post" action="#" onsubmit="return false;">
             <h1>FkBoard</h1>
             <div class="input-group">
-                <div class="input"><label for="address">Adresse</label><input type="text" name="address" id="address" placeholder="localhost" required></div>
-                <div class="input"><label for="port">Port</label><input type="number" name="port" id="port" placeholder="50000" required></div>
+                <div class="input">
+                    <label for="address">Adresse</label
```

Comment:
The closing tag of an HTML element without attribute should be on the same line.